### PR TITLE
[easy] skipIfTorchInductor - use condition variable

### LINKED ISSUE #112465: [tests] skipIfTorchInductor is broken
### 🐛 Describe the bug

condition arg is unused

Hence skipRocmIfTorchInductor is broken

### Versions

main

cc @ezyang @msaroufim @wconstab @bdhirsh @anijain2305 @zou3519 @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler

### PR DESCRIPTION
**Pull request by davidberard98**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #112774

Fixes #112465

### CODE REVIEW

**Review by jon-chuang — APPROVED** [review]
LGTM, but will need a core maintainer approval

**Review by aaronenyeshi — APPROVED** [review]
LGTM

**Pull request closed without merging**
